strapi/strapi
# [Marketplace] Add sort menu to the marketplace page

### How to test it
1. Go to the marketplace page
2. Click on the "Sort by" button, then click on "Newest", then it should start loading and show the plugin list order by submission date, the url should change too with the new query params
3. Click again on the "Sort by" button, then click on "Alphabetical order", it should be the same initial state
4. Do the same in the providers tab
